hyperspy/drawing/utils.py

```diff
@@ -580,6 +577,46 @@ def plot_images(images,
               if sig.axes_manager.navigation_size > 0
               else 1)
 
+    # If no cmap given, get default colormap from pyplot:
+    if cmap is None:
+        cmap = [plt.get_cmap().name]
+    elif cmap == 'mpl_colors':
+        for n_color, c in enumerate(mpl.rcParams['axes.prop_cycle']):
+            make_cmap(colors=['#000000', c['color']],
+                      name='mpl{}'.format(n_color))
+        cmap = ['mpl{}'.format(i) for i in
+                    range(len(mpl.rcParams['axes.prop_cycle']))]
+    # cmap is list, tuple, or something else iterable (but not string):
+    elif hasattr(cmap, '__iter__') and not isinstance(cmap, str):
```

Comment:
You could pass many thing in there and get an error later in the code. I would rather consider all the expected input and finished in the `else` by raising a ValueError when the parameter is not correct to provide a clear error message.